### PR TITLE
Ability to specify mountpoint directory mode

### PR DESCRIPTION
**Pull request by eagleusb**
It permits to specify the directory mode for the mountpoint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +42,7 @@ disk_additional_disks:
 * `mount` is the directory where the new disk should be mounted.
 * `user` sets owner of the mount directory (default: `root`).
 * `group` sets group of the mount directory (default: `root`).
+* `mode` sets the mountpoint directory permissions (default: `0775`).
```
Comment (MiLk) [inline README.md]: Could we make the default `755`?

**Review by MiLk — COMMENTED** [review]
Thanks for your contribution!
I just left a small comment.

File: README.md (hunk 1 above)
Comment (eagleusb) [reply README.md]: Of course, my bad for that permissive default 😸

**Comment by MiLk** [comment]
Thanks! Sorry for the delay.

**Pull request merged**